NEWS.adoc

```diff
@@ -1,10 +1,57 @@
 = Freedoom project news
 
-== HEAD
-=== Graphics
-  * New ESPI* textures are included as an alternative to the STAR*
-    textures.  These are named after and in honor of Esa Repo (Espi).
+== 0.13.0 (2023-12-10)
+=== General
+  * Improved vanilla compatibility.
+  ** Boom features removed.
+  ** Hall of mirrors greatly reduced.
+  ** Visplane overflows fixed.
+  ** Savegame buffer overflow errors remain.
+
+=== Levels
+  * Relevant Eureka warnings fixed.
+  * New levels E1M9, E2M2, E2M3, E2M4, E2M7, E2M8, E3M5, MAP07, MAP21 and MAP27.
+  * Various level renames.
+  * Numerous vanilla fixes and aesthetic modernizations.
+  * Fixed and standardized secret exits.
+
+=== Manual
+  * A Spanish manual.
```

Comment:
* Spanish and French manuals.

Reply:
This is now "French and Spanish translations." I figured it was in the "Manual" section anyway.